cli/filplus.go

```diff
@@ -799,3 +801,149 @@ var filplusSignRemoveDataCapProposal = &cli.Command{
 		return nil
 	},
 }
+
+var filplusListAllAllocationsCmd = &cli.Command{
```

Comment:
Not really suggestions, just thoughts inviting discussion: These could be integrated into `list-allocations` and `list-claims` just by making the client optional. Also as they are, I'm not sure the `--expired` makes as much sense for these `all` listings as stand-alone commands.

Reply:
I has a similar initial though but I wasn't sure if that would make things easier or more tricky. I will update the CLI as per your suggestion.

Reply:
I have updated the CLI and added a JSON flag to allow json output as well.